packages/design-tokens/tsconfig.cjs.json

```diff
@@ -2,7 +2,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "module": "CommonJS",
-    "target": "es5",
+    "target": "es2017",
```

Comment:
For cjs bundles, we're going the opposite direction and targeting a newer ES version. This will stop transpiling es6 syntax, supported in all our target browsers.